lib/log.js

```diff
@@ -78,23 +79,34 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function (event, request, data) {
+Lug.prototype.activityEvent = function (event, request, sessionToken, data) {
```

Comment:
I wonder, if we can pull `sessionToken` out of `request.credentials` rather than passing it in, or if that would be unreliable in some cases.